language/de/extimgaslink.php

```diff
@@ -23,5 +23,11 @@
 }
 
 $lang = array_merge($lang, array(
+	'EXTIMGASLINK_CONFIG'			=> 'Erlaubte Herkunft von in Beiträgen verwendeten Bildern',
+	'EXTIMGASLINK_CONFIG_EXPLAIN'	=> 'Alle anderen Bilder werden durch Links ersetzt.',
+	'EXTIMGASLINK_INVALID_CONFIG'	=> 'Die angegebene Bildherkunft ist ungültig.',
+	'EXTIMGASLINK_SECURE_SITES'		=> 'nur Bilder von sicheren Webseiten und dem Server',
+	'EXTIMGASLINK_SERVER_ONLY'		=> 'nur Bilder vom Server',
```

Comment:
Ach so, "von in Beiträgen verwendet... " ist vielleicht auch besser ohne "von ", also 'Erlaubte Herkunft in Beiträgen verwendeter Bilder'

Oder noch besser einfach 
> Zulässige Bilder 

Und eine etwas ausführlichere Beschreibung drunter. Dann passt es vom Stil her besser.